lib/static/modules/selectors/index.js

```diff
@@ -0,0 +1,5 @@
+// There is a bug https://github.com/babel/babel-eslint/issues/167
```

Comment:
It looks like that the problem described in this issue is already fixed am I right? Maybe we should update `babel-eslint`?